opencga-app/app/scripts/azure/arm/mongodb/postdeploy.sh

```diff
@@ -31,12 +31,47 @@ installDeps () {
     sleep 15
 }
 
+
+scanForNewDisks() {
+    # Looks for unpartitioned disks
+    DEVS=($(fdisk -l 2>/dev/null | egrep -o '(/dev/[^:]*):' | egrep -o '([^:]*)'))
+
+    for DEV in "${DEVS[@]}";
+    do
+        isPartitioned ${DEV}
+    done
+}
+
+isPartitioned() {
+    OUTPUT=$(partx -s ${1} 2>&1)
+    if [[ $OUTPUT == *"failed to read partition table"* ]]; then
+        # found unpartitioned disk
+        formatAndMountDisk "${DEV}"
+    fi
+}
+
+formatAndMountDisk() {
+    #partitions primary linux partition on new disk
+    echo 'type=83' | sudo sfdisk ${1}
+    mkfs -t ext4 ${1}1
+    mkdir /datadrive
+    mount ${1}1 /datadrive
+    fs_uuid=$(blkid -o value -s UUID ${1}1)
+    echo "UUID=${fs_uuid}   /datadrive   ext4   defaults,nofail   1   2" >> /etc/fstab
+}
+
 generateCertificate() {
     sed -i -e 's/# server_names_hash_bucket_size 64/server_names_hash_bucket_size 128/g' /etc/nginx/nginx.conf
     certbot --nginx -d ${APP_DNS_NAME} -m ${CERT_EMAIL} --agree-tos -q
     nginx -t && nginx -s reload
     cat /etc/letsencrypt/live/${APP_DNS_NAME}/privkey.pem /etc/letsencrypt/live/${APP_DNS_NAME}/cert.pem > /etc/ssl/mongo.pem
     cat /etc/letsencrypt/live/${APP_DNS_NAME}/chain.pem >> /etc/ssl/ca.pem
+
+    #configure mongodb to use /datadrive for storage
+    mkdir /datadrive/mongodb
+    chmod 777 -R /datadrive
```

Comment:
This is fine for now, we might want to consider a more appropriate ACL though

Reply:
Added ACL in https://github.com/opencb/opencga/pull/1109/commits/95f53f898d4b00259d3478e5c995113729fd61b2